### DISCUSSION TITLE
Removing blank year of contributions

### DISCUSSION THREAD

**Discussion by danielopq** [community/community#114258] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all,

I opened my account on github a few years ago, but I didn't start to be actually active until the past year. If you click on my contributions, you can see it. I was wondering if there is any way to remove/hide those years in blank. Thanks

**Comment by MandoIsGood** [comment]
Unfortunately, GitHub doesn't currently offer a way to directly remove or hide inactive years from your contribution graph.

**Reply by danielopq** [reply]
I see. Thanks
